mlflow/transformers/__init__.py

```diff
@@ -532,12 +534,20 @@ def save_model(
             inference_config=inference_config,
         )
 
+    model_name = transformers_model.model.name_or_path
+
     # Get the model card from either the argument or the HuggingFace marketplace
-    card_data = model_card if model_card is not None else _fetch_model_card(transformers_model)
+    card_data = model_card if model_card is not None else _fetch_model_card(model_name)
 
     # If the card data can be acquired, save the text and the data separately
     _write_card_data(card_data, path)
 
+    # Retrieve license information from the huggingface_hub repository if it exists
+    license_text = _fetch_license(model_name, card_data)
+
+    # Write the license information (or guidance) along with the model
+    _write_license_information(license_text, path)
```

Comment:
Why do we need to write the text into LICENSE.txt file instead of directly copying the file over to the path?

Reply:
+1, changing file extension to `.txt` may be confusing as well.

Reply:
Update the logic to copy the file as-is just in case for non text licenses